composer.json

```diff
@@ -29,6 +29,7 @@
         "doctrine/orm": "^2.3",
         "friendsofphp/php-cs-fixer": "^2.8",
         "friendsoftwig/twigcs": "^3.1.2",
+        "symfony/expression-language": "^5.1@dev",
```

Comment:
This should use a range. The tests are failing because, when we test using PHP 7.1 (which is incompatible with Symfony 5), this dependency cannot be installed at this version. Use the range `^4.4|^5.0`.

Reply:
Done for 4.4/